.github/workflows/release.yml

```diff
@@ -2,16 +2,16 @@ name: Release
 on: [workflow_dispatch] # Manual trigger
 jobs:
   build:
-    runs-on: ubuntu-20.04
-    container:
-      image: openjdk:17-jdk
-      options: --user root
+    runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v1
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+            distribution: temurin
+            java-version: 17
+      - uses: gradle/wrapper-validation-action@v1
       - run: ./gradlew build publish --stacktrace
         env:
-          MAVEN_URL: ${{ secrets.MAVEN_URL }}
-          MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.MAVEN_PASSWORD }}
\ No newline at end of file
+            MAVEN_URL: ${{ secrets.MAVEN_URL }}
+            MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
+            MAVEN_PASSWORD: ${{ secrets.MAVEN_PASSWORD }}
```

Comment:
unnecessary whitespace changes

Reply:
I was going to say something about consistency between projects, but nope! i probably just screwed up here; Will fix

Reply:
Fixed!